sdcm/nemesis.py

```diff
@@ -3683,6 +3683,12 @@ def disrupt_remove_node_then_add_node(self):  # pylint: disable=too-many-branche
         if self._is_it_on_kubernetes():
             raise UnsupportedNemesis("On K8S nodes get removed differently. Skipping.")
 
+        if SkipPerIssues('https://github.com/scylladb/scylladb/issues/21815', params=self.tester.params):
+            # TBD: To be removed after https://github.com/scylladb/scylladb/issues/21815 is resolved
+            context_manager = ignore_stream_mutation_fragments_errors
+        else:
+            context_manager = contextlib.nullcontext()
```

Comment:
name might be a bit better, also you initialize this one twice, it won't work
```suggestion
            ignore_issues_cm = contextlib.nullcontext
```